### DISCUSSION TITLE
Plan to support graphql-java 22?

### DISCUSSION THREAD

**Discussion by FutureGadget** [Netflix/dgs-framework#1901] [Q&A] [answered]
https://github.com/graphql-java/graphql-java/issues/3586
When I benchmarked, graphql-java 22 has huge performance gain when the size of the object before response is large.
Do you have a plan to support graphql-java 22 in the near future?
Thanks!

**Comment by srinivasankavitha** [comment, marked as answer]
Yes, we will be upgrading in the next release or two releases out.
